docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
I was thinking `group` first, but finally `stack` might be better. Can we explicitly document this `dataset.stack` as an ID and not a name to avoid any confusion with label/text/title?